e2e/test/scenarios/custom-column/custom-column.cy.spec.js

```diff
@@ -684,7 +684,8 @@ describe("scenarios > question > custom column", () => {
     cy.focused().should("have.attr", "class").and("eq", "ace_text-input");
   });
 
-  it("should render custom expression helper near the custom expression field", () => {
+  // TODO: fixme!
+  it.skip("should render custom expression helper near the custom expression field", () => {
```

Comment:
Created a follow-up issue so this doesn't get lost:
https://github.com/metabase/metabase/issues/38258